packages-common/pluggable-widgets-generator/generators/app/templates/packages/package_web.json.ejs

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
`package_web` and `package_native` are very similar. Maybe just have a single one with some templating to put native or web string  there?

Reply:
I think combining everything will make the template complex and badly readable.